frontend/package.json

```diff
@@ -42,16 +44,15 @@
     "styled-components": "^5.3.6",
     "swr": "2.0.3",
     "web-vitals": "^2.1.4",
-    "yup": "^0.29.2",
```

Comment:
Why did you remove these dependencies ??
Am sure they are used some where in the project